connaisseur/exceptions.py

```diff
@@ -55,3 +55,25 @@ class UnknownVersionError(Exception):
 
 class AmbiguousDigestError(BaseConnaisseurException):
     pass
+
+
+class AlertingException(Exception):
+
+    message: str
+    status_code: int
+
+    def __init__(self, message: str):
+        self.message = message
+        self.status_code = 500
```

Comment:
Is this `status_code` ever used?

Reply:
I couldn't find any usage :+1: removed it!